packages/central-server/app/admin/patientMerge/mergePatient.js

```diff
@@ -346,6 +354,12 @@ export async function reconcilePatientFacilities(models, keepPatientId, unwanted
   return newPatientFacilities;
 }
 
+export async function refreshMultiChildRecordsForSync(model, records) {
```

Comment:
I see why you brought this out, but honestly I don't think it's worth it. Adds another concept to understand vs. just having three for loops in a row